engineering_tools/opendrive_lanelet2_converter/opendrive2lanelet2convertor.py

```diff
@@ -1,283 +0,0 @@
-        xml.SubElement(relation_element, "tag", {"k": "near_spaces", "v": "[]"})
-        xml.SubElement(relation_element, "tag", {"k": "turn_direction", "v": self.turn_direction})
-        return relation_element
-
-    # start is the first point of lanelet
-    # mid is the secound point of lanlelet
-    # end is the last point of lanelet
-    # set_direction is used by set_turn direction to check if the lanelet is turning left or right
-    def set_direction(self, start, mid, end):
-        if(((mid[0] - start[0])*(end[1] - start[1]) - (mid[1] - start[1])*(end[0] - start[0])) > 0):
-            return 1
-        else:
-            return 0
-
-    #set_turn_direction is used to set set the turn_direction tag in lanelet
-    def set_turn_direction(self):
-        turn_direction = "straight"
-        if(len(self.from_cad_id) == 1 and len(self.to_cad_id) == 1):
-            start = np.array([self.member_left.nodes[0].local_x,self.member_left.nodes[0].local_y])
-            mid = np.array([self.member_left.nodes[1].local_x,self.member_left.nodes[1].local_y])
-            end = np.array([self.member_left.nodes[-1].local_x,self.member_left.nodes[-1].local_y])
-
-            start_mid = mid - start
-            start_end = end - start
-            cosine_angle = np.dot(start_mid, start_end) / (np.linalg.norm(start_mid) * np.linalg.norm(start_end))
-            alpha = np.arccos(cosine_angle)
-
-            if(np.degrees(alpha) > 10):
-                if(self.set_direction(start,mid,end) == 0):
-                    turn_direction = "right"
-                else:
-                    turn_direction = "left"
-
-        self.turn_direction = turn_direction
-
-# class used to convert opendrive map to lanelet2 map
-class Opendrive2Lanelet2Convertor:
-    def __init__(self, fn):
-        self.scenario, self.geoReference = self.open_drive_loader(fn)
-        self.root = xml.Element('osm',{'generator': 'lanelet2','version': '0.6'})
-        self.nodes = xml.Element('nodes')
-        self.ways = xml.Element('ways')
-        self.relations = xml.Element('relations')
-        self.all_nodes = []
-        self.all_ways = []
-        
-    def open_drive_loader(self, fn):
-        fi = open(fn.format(os.path.dirname(os.path.realpath(__file__))), "r")
-        tree = etree.parse(fi)
-        root = tree.getroot()
-        geoReference = root[0][0].text
-        open_drive = parse_opendrive(root)
-        road_network = Network()
-        road_network.load_opendrive(open_drive)
-        return road_network.export_commonroad_scenario(), geoReference
-
-    # convert x,y values to geo points using the geo reference defined in the input file
-    def get_point_geo(self,x,y):
-        inProj = Proj(self.geoReference)
-        return inProj(x,y,inverse=True)
-
-    def write_xml_to_file(self,fn):
-                
-        geoRefXml = xml.Element('geoReference')
-        geoRefXml.text = self.geoReference
-        self.root.append(geoRefXml)
```

Comment:
This is the line needed to write geoRereference into osm file while converting. 
The file is entirely deleted in this branch to have minimum conflict with develop.
However, this script will be added into separate library in the near future.